geocomp/common/binary_search_tree.py

```diff
@@ -45,243 +44,240 @@ def __nonzero__(self):
     def __bool__(self):
         return False
 
+    def is_red(self) -> bool:
+        return False
+
 class BinarySearchTree:
     def __init__(self):
         self.root = NilNode.instance()
         self.size = 0
         self.control = {}
 
-    def __str__helper(self, node, level = 0, indent = "   "):
-        s = level * indent + str(node.id)
+    def __str__helper(self, node: Node, level: int = 0, indent: str = "   "):
```

Comment:
```suggestion
    def __str_helper(self, node: Node, level: int = 0, indent: str = "   "):
```
Senão fica parecendo um dunder?